### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by Jaesin**
This adds the tests in ui-tests as well as a template file for the add list which wouldn't be available in 8.0.x.

### CODE REVIEW

File: content_entity_base.module [hunk 1]
```diff
@@ -20,5 +20,42 @@ function content_entity_base_help($route_name, RouteMatchInterface $route_match)
...
+      return '';
+  }
+}
+
+/**
+ * Implements hook_theme_registry_alter.
+ */
+function content_entity_base_theme_registry_alter(&$theme_registry) {
+  // Check for entity_add_list which was added in 8.1.x.
+  if (!isset($theme_registry['entity_add_list'])) {
```
Comment (dawehner) [inline content_entity_base.module:32]: Do we really still want to support 8.0.x ?

Comment (Jaesin) [inline content_entity_base.module:32]: I have to deal with some installs that have yet to upgrade to 8.1.x. =( 

I doubt I am the only one.

Comment (Jaesin) [inline content_entity_base.module:32]: That said, I'm on the fence about it. It would be nice to hear from others.

**Comment by Jaesin** [comment]
This last commit fixes most of the issues exposed by tests so far except `administer $entity_type_id` permission not giving sufficient access. That seems like another issue to me.

**Comment by Jaesin** [comment]
Well uh except in 8.1.x. =(

**Comment by dawehner** [comment]
Nice work!

**Pull request merged**